pkg/analyzer/analyzer.go

```diff
@@ -0,0 +1,140 @@
+package analyzer
+
+import (
+	"os"
+	"path/filepath"
+	"regexp"
+	"sync"
+
+	"github.com/Checkmarx/kics/internal/metrics"
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog/log"
+)
+
+// openAPIRegex - Regex that finds OpenAPI defining property
+// cloudRegex - Regex that finds Cloud Formation defining property
+// k8sRegex - Regex that finds Kubernetes defining property
+var (
+	openAPIRegex = regexp.MustCompile("(\\s*\"openapi\":)|(\\s*openapi:)")
+	cloudRegex   = regexp.MustCompile("(\\s*\"Resources\":)|(\\s*Resources:)")
```

Comment:
This is the only way to identify cloud formation file? I'm asking this, because if, for some reason, we have others resources files containing `Resource` keyword, it would be a problem, no?

Reply:
I think it needs to be uppercase R. But we can also test this: 
https://github.com/aws-cloudformation/aws-cloudformation-template-schema/blob/master/src/main/resources/Schema.template#L371

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resources-section-structure.html

Reply:
It's too bad that `Resources` is the only required field: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-anatomy.html  But we can try find a pattern that matches the any resources inside the `Resources:` key.

Reply:
I will add more cases to unit tests for testing purposes